event-listener/src/test/java/dasniko/keycloak/events/LastLoginTimeListenerTest.java

```diff
@@ -24,14 +25,19 @@ public class LastLoginTimeListenerTest extends TestBase {
 	@Container
 	private static final KeycloakContainer keycloak = new KeycloakContainer()
 		.withRealmImportFile("demo-realm.json")
+		.withNightly()
 		.withEnv("KC_SPI_EVENTS_LISTENER_LAST_LOGIN_TIME_ATTRIBUTE_NAME", "lastLogin")
 		.withProviderClassesFrom("target/classes");
 
 	@Test
-	@Disabled
 	public void testLastLoginTime() {
 		Keycloak admin = keycloak.getKeycloakAdminClient();
 
+		// enable unmanaged attributes to be able to fetch the custom user attribute
```

Comment:
The different behavior was introduced by https://github.com/keycloak/keycloak/issues/39595.